web/templates/status_emoji.hbs

```diff
@@ -6,7 +6,7 @@
 {{else if url}}
 {{!-- note that we have no still_url --}}
 <img src="{{url}}" class="emoji status-emoji" data-animated-url="{{url}}" />
-{{else if emoji_code}}
-<span class="emoji status-emoji emoji-{{emoji_code}}"></span>
+{{else if emoji_name}}
+<span class="emoji status-emoji emoji-{{emoji_code}}" id="status-emoji-name" data-tippy-content=":{{emoji_name}}:"></span>
```

Comment:
Let's handle `status-emoji-name` as a class; an `id` should only appear once in the DOM.